packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.story.js

```diff
@@ -69,7 +69,7 @@ storiesOf('Components|Inputs', module)
               }
               defaultExpandMultilineText={defaultExpandMultilineText}
               isAutofocussed={boolean('isAutofocussed', false)}
-              cacheMeasurements={boolean('cacheMeasurements', true)}
+              cacheMeasurements={boolean('cacheMeasurements', false)}
```

Comment:
@CarlosCortizasCT  Do you typically set the default value in storybook to false even if the default value in the component is true?

Reply:
I think it's better to use `false` here because, otherwise, the `isCondensed` check won't work and users won't know why.